.circle/pytests.sh

```diff
@@ -10,25 +10,10 @@ set -x         # Print command traces before executing command.
 
 code=0
 
-if [ "${CIRCLE_NODE_TOTAL:-1}" == "1" ]; then
-    docker run -ti --env SAVE_CIRCLE_ARTIFACTS="/scratch" -v ${SCRATCH}/py3:/scratch -w /scratch --entrypoint=/usr/local/miniconda/bin/py.test niworkflows:py3 /root/niworkflows -n ${CIRCLE_NPROCS:-4} -v --junit-xml=/scratch/pytest.xml && \
-    docker run -ti --env SAVE_CIRCLE_ARTIFACTS="/scratch" -v ${SCRATCH}/py2:/scratch -w /scratch --entrypoint=/usr/local/miniconda/bin/py.test niworkflows:py2 /root/niworkflows -n ${CIRCLE_NPROCS:-4} -v --junit-xml=/scratch/pytest.xml
-    code=$(( $code + $? ))
-else
-    case ${CIRCLE_NODE_INDEX} in
-    0)
-        docker run --rm=false -ti --env SAVE_CIRCLE_ARTIFACTS="/scratch" --env CIRCLE_NPROCS=4 -v ${SCRATCH}/py3:/scratch -w /scratch --entrypoint=/usr/local/miniconda/bin/py.test niworkflows:py3 /root/niworkflows  -n 2 -v --junit-xml=/scratch/pytest.xml
-        code=$(( $code + $? ))
-        ;;
-    1)
-        docker run --rm=false -ti --env SAVE_CIRCLE_ARTIFACTS="/scratch" --env CIRCLE_NPROCS=4 -v ${SCRATCH}/py2:/scratch -w /scratch --entrypoint=/usr/local/miniconda/bin/py.test niworkflows:py2 /root/niworkflows  -n 2 -v --junit-xml=/scratch/pytest.xml
-        code=$(( $code + $? ))
-        ;;
-    esac
-fi
+docker run -ti --env SAVE_CIRCLE_ARTIFACTS="/scratch" -v ${SCRATCH}/py3:/scratch -w /scratch --entrypoint=/usr/local/miniconda/bin/py.test niworkflows:py3 /root/niworkflows -n ${CIRCLE_NPROCS:-4} -v --junit-xml=/scratch/pytest.xml
+code=$(( $code + $? ))
```

Comment:
I find the whole exit code thing here a little strange. We have `set -e` in the preamble, so if `$?` is non-zero, we won't even get to this line. I think we can drop the whole `code` variable altogether.

@oesteban I think you wrote this script. Am I missing something?

Reply:
Yes, that's right. Anyways it makes less sense now.